login-workflow/example/src/app/pages/auth/auth.component.ts

```diff
@@ -70,7 +73,11 @@ import { Router } from '@angular/router';
 
         <!-- Custom Create Account page -->
         <ng-template #createAccountViaInvitePage>
-            <pxb-create-account-invite #createAccountInviteVC [accountDetails]="accountDetails">
+            <pxb-create-account-invite
+                #createAccountInviteVC
+                [accountDetails]="accountDetails"
+                [existingAccountSuccessScreen]="registrationSuccessScreen"
```

Comment:
It's a little confusing having these both use the same success screen. It makes it hard to tell if I'm seeing the existing account success screen or the regular one.

Reply:
There's now a unique success screen for whenever an account has been previously registered.  

Currently, the 'create-account' 'already-registered' success screen shows the default.  
The 'create-account-invite' 'already-registered' success screen shows the customized one.  

The updates are hosted here: https://pxblue-temp.web.app/

Reply:
There's now a unique success screen for whenever an account has been previously registered.  

Currently, the 'create-account' 'already-registered' success screen shows the default.  
The 'create-account-invite' 'already-registered' success screen shows the customized one.  

The updates are hosted here: https://pxblue-temp.web.app/